tmt/steps/execute/run.sh

```diff
@@ -140,11 +159,16 @@ tmt_run_test () {
     tmt_verbose 2 "type: ${tmt_TYPE}"
 
     tmt_run_${tmt_TYPE} "$path" "$environment" "$cmd"
-    echo "$?" >"$tmt_LOGCODE_F"
+    local exitcode="$?"
+    echo "$exitcode" >"$tmt_LOGCODE_F"
 
-    grep -q '^0$' "$tmt_LOGCODE_F" \
-        && echo -n "." \
-        || echo -n "F"
+    [[ "$exitcode" == "0" ]] && {
+        echo -n "."
+        result='pass'
+    } || {
```

Comment:
It's in no way blocking, nor should it change the behaviour, but just to be sure, in case someone changes the result line to something else, I would add:
```suggestion
        result='pass'
        :
    } || {
```

So the following block will not be executed  if the condition was already true.

Reply:
Results handling functionality moved to the `simple` execute plugin.